src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -61,6 +61,25 @@ class PipelineInfo:
     failures: str
 
 
+@dataclass
+class AWSInstanceProfileInfo:
```

Comment:
This file size is getting out of control, could you put the aws instance profiles related code into aws_instance_profiles.py in the same folder? :) we discussed it last week

Reply:
Makes sense